config/cmake/addfmt.cmake

```diff
@@ -13,15 +13,15 @@
 
 option(${PROJECT_NAME}_USE_EXTERNAL_FMT "Use external copy of {fmt}" OFF)
 mark_as_advanced(${PROJECT_NAME}_USE_EXTERNAL_FMT)
-
+if (NOT TARGET fmt::fmt)
 if(${PROJECT_NAME}_USE_EXTERNAL_FMT)
```

Comment:
It looks like this block of code needs to be reformatted to go a level deeper.

Reply:
why is the precommit not catching that?